orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/pipeline/servergroup/strategies/RedBlackStrategy.groovy

```diff
@@ -80,6 +84,18 @@ class RedBlackStrategy implements Strategy, ApplicationContextAware {
       )
     }
 
+    if(stageData?.waitTime) {
+      def waitContext = [waitTime: stageData?.waitTime]
+      stages << newStage(
+        stage.execution,
+        waitStage.type,
+        "wait",
+        waitContext,
+        stage,
+        SyntheticStageOwner.STAGE_AFTER
```

Comment:
I believe we don't let you shrinkCluster to 1 (otherwise it's just highlander) so this seems reasonable to have injected after the <optional> shrinkCluster.